.github/workflows/translations.yml

```diff
@@ -49,16 +49,13 @@ jobs:
           declare -r base_files=$(find src/main/resources/resources/logisim/* -name "*.properties" | grep -E '*/[a-zA-Z]+.properties$' | grep -v settings.properties)
 
           # Read currently supported languages from settings.properties
-          declare -r langs_list="$(cat src/main/resources/resources/logisim/settings.properties | cut -d  ' ' -f3-) "
```

Comment:
I think that instead of `$(cat FILE | cut ...)` we can just do `$(cut .... < FILE)`